content/desktop/hardened-desktop/settings-management/configure.md

```diff
@@ -135,6 +147,9 @@ The following `admin-settings.json` code and table provides an example of the re
 |`analyticsEnabled`|  |If `value` is set to false, Docker Desktop doesn't send usage statistics to Docker. |
 |`extensionsEnabled`|  |If `value` is set to false, Docker extensions are disabled. |
 |`scout`|| Setting `useBackgroundIndexing` to `false` disables automatic indexing of images loaded to the image store. Setting `sbomIndexing` to `false` prevents the manual indexing triggered by inspecting an image in Docker Desktop.<br><br>**Note**: Users can still use the `docker scout` CLI commands to index images, even if indexing is disabled in Settings Management. |
+| `allowExperimentalFeatures`| | If `locked` is set to `true`, users are no longer able to access or use experimental features.|
```

Comment:
```suggestion
| `allowExperimentalFeatures`| | If `value` is set to `false`, the experimental features are disabled.|
```

Reply:
As with the other admin settings, here we only need to describe what the setting controls, as the locking behaviour is described at the top of the page, line 46